### PR TITLE
Integrate wikid into NEL benchmark

### PR DESCRIPTION
**Pull request by rmitsch**
## Description

Integrate `wikid` for Wiki parsing and KB generation (the branch name is a misnomer).

### Types of change

Enhancement.

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Review by polm — COMMENTED** [review]
This looks fine, there's a few small points with dependencies but after those are fixed I think this should be good to go. It also ran fine for me locally.

Also it's not new to this PR, but as part of running this I confirmed that the `download_mewsli9` command doesn't run correctly twice in a row - it fails moving files around if you try to run it twice. I'm not sure if the outputs need to be adjusted, or if it would be easier to have it remove any existing files to avoid errors partway through the process. That can be addressed outside this PR if you prefer.

**Comment by rmitsch** [comment]
Should be addressed - give it another try?

File: benchmarks/nel/scripts/train.sh [hunk 1]
```diff
@@ -1,20 +1,22 @@
 #!/bin/bash
```
[the remark this answers is not in the document]
Comment (polm) [reply benchmarks/nel/scripts/train.sh:1]: No worries, that's a good enough reason. Just was curious.

**Comment by polm** [comment]
Thanks, I was able to confirm this works!

**Review by polm — APPROVED** [review]

**Pull request merged**